ext/digest/digest.c

```diff
@@ -537,6 +537,36 @@ rb_digest_class_init(VALUE self)
  *		Data_Wrap_Struct(0, 0, 0, (void *)&sha1));
```

Comment:
Would be good to update the documentation too?

Reply:
Yes.
This should have been updated when `rb_digest_make_metadata` was added.